### ISSUE TITLE
Open ended range.

### ISSUE THREAD

**Issue by adjenks** [mengxiong10/vue2-datepicker#548]
Is it possible to allow only one side of the range to be entered?
I want to allow ranges without both ends. Can you think of a way to rig the existing system to do this?

**What problem does this feature solve?**
Open ended ranges.

**What does the proposed API look like?**
Something like this?
`:open-range="true"`

I might just just create a menu of buttons that say "before", "after", "between" and "on". To do open ranges, closed range, and single date. But I'd like to just allow one end to be null.

**Comment by mengxiong10** [comment]
You can use two components: https://codesandbox.io/s/date-range-optional-range-z06gv.

You can customize range component: https://codesandbox.io/s/range-in-datepicker2x-mode-hlzrg?file=/src/App.vue

**Comment by adjenks** [comment]
In the end, I used the side panel to toggle different modes, I didn't really have the time to understand customizing the range component and I didn't really like the dual component approach but maybe I would have used two components if I did it again. Ideally it would just have a "nullable-range" option though. Feel free to add that feature, or not. Haha. It's a great component either way though, thank you for the awesome work.
![image](https://user-images.githubusercontent.com/10687569/100292046-42c5db80-2f34-11eb-81a7-cca6b0232398.png)
